crypto/sha256.c

```diff
@@ -249,6 +249,10 @@ void SHA256_Final(uint8_t *digest, SHA256_CTX *ctx)
     uint32_t last, padn;
     uint32_t high, low;
     uint8_t msglen[8];
+#ifdef FORCEL32_NOT_SUPPORTED
+    uint8_t sha256_padding_ram[64];
```

Comment:
You're using `PADDING_stack` in the other file, suggest using the same suffix in both places.